integration-tests/suites/connections_and_endpoints.go

```diff
@@ -147,36 +144,3 @@ func (s *ConnectionsAndEndpointsTestSuite) TestConnectionsAndEndpoints() {
 	}
 
 }
-
-func endpointComparison(endpoint1 types.EndpointInfo, endpoint2 types.EndpointInfo) bool {
```

Comment:
Makes more sense to move this to somewhere that can be called by multiple test suites.